ent/schema/repo.go

```diff
@@ -35,6 +35,9 @@ func (Repo) Fields() []ent.Field {
 		// Denormalization to sort with deployment.
 		field.Time("latest_deployed_at").
 			Optional(),
+		// The 'owner_id' field is the ID who activated the repository.
+		field.Int64("owner_id").
+			Optional(),
```

Comment:
Save the owner and use the token of the user when Gitploy deploys for an auto-deployment.